### PR TITLE
Ability to specify skip tests

### PR DESCRIPTION
**Pull request by leodotcloud**
https://github.com/rancher/rancher/issues/23905

### CODE REVIEW

**Comment by cjellick** [comment]
why is build failing

**Comment by leodotcloud** [comment]
@cjellick had to get rid of a comment. Re-pushed the changes

**Comment by leodotcloud** [comment]
@cjellick it's green now

**Comment by dramich** [comment]
@leodotcloud There is mix use of CIS vs Cis, can you standardize?

**Comment by leodotcloud** [comment]
@dramich If I use all uppercase letters, I ran into a problem with the generator.

File: apis/management.cattle.io/v3/cluster_scan_types.go [hunk 1]
```diff
@@ -11,9 +11,29 @@ import (
...
+	DefaultNamespaceForCis     = "security-scan"
+	DefaultSonobuoyPodName     = "security-scan-runner"
+	ConfigMapNameForUserConfig = "security-scan-cfg"
+
+	RunCISScanAnnotation         = "field.cattle.io/runCisScan"
+	SonobuoyCompletionAnnotation = "field.cattle.io/sonobuoyDone"
+	CisHelmChartOwner            = "field.cattle.io/clusterScanOwner"
 )
 
+type CisScanConfig struct {
```
Comment (alena1108) [inline apis/management.cattle.io/v3/cluster_scan_types.go]: should it be CIS?

Comment (alena1108) [reply apis/management.cattle.io/v3/cluster_scan_types.go]: Or the other way around. If generator is unhappy with the caps, we should switch to camel case in other places.

**Comment by leodotcloud** [comment]
Filed this issue for the problem when using uppercase for the word CIS. https://github.com/rancher/rancher/issues/24345

**Comment by alena1108** [comment]
LGTM

**Pull request merged**